appveyor.yml

```diff
@@ -2,59 +2,35 @@ version: "{branch}-ci-{build}"
 os: Visual Studio 2015
 
 environment:
-  RUBY_VERSION: 200
   matrix:
-  - build: mingw
-    platform: x86
+  - CMAKE_GENERATOR: 'MSYS Makefiles'
     MINGW_ARCH: i686
-    MSYSTEM: MINGW32
     MINGW_ROOT: C:\msys64\mingw32
     BOOST_ROOT: C:\msys64\mingw32
     BOOST_LIBRARYDIR: C:\msys64\mingw32\lib
     BOOST_INCLUDEDIR: C:\msys64\mingw32\include\boost
-    CMAKE_GENERATOR: 'MSYS Makefiles'
-  - build: mingw
-    platform: x64
+    QT_DIR: C:\Qt\5.8\mingw53_32
+  - CMAKE_GENERATOR: 'MSYS Makefiles'
     MINGW_ARCH: x86_64
-    MSYSTEM: MINGW64
     MINGW_ROOT: C:\msys64\mingw64
     BOOST_ROOT: C:\msys64\mingw64
     BOOST_LIBRARYDIR: C:\msys64\mingw64\lib
     BOOST_INCLUDEDIR: C:\msys64\mingw64\include\boost
```

Comment:
Appveyor has only 32-bit mingw-compiled qt libs preinstalled. https://www.appveyor.com/docs/build-environment/#qt
AFAIK it is not possible to link VC-compiled Qt with Mingw-compiled cucumber, or use 32-bit  mingw-compiled lib.